@xen-orchestra/lite/src/components/NoData.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="noData">
+    <img alt="No data" class="img" src="../assets/undraw-bug-fixing.svg" />
```

Comment:
It would be preferable to use the root `@` alias. (BTW, I've seen some relative paths across the project that will need to be updated)

```suggestion
    <img alt="No data" class="img" src="@/assets/undraw-bug-fixing.svg" />
```

Also, maybe we should change the file name to be more meaningful?